crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -230,10 +293,13 @@ where
 
         let vertex_buffer_layout = layout.get_layout(&vertex_attributes)?;
 
-        // The fragment shader is only used when the normal prepass is enabled or the material uses an alpha mask
+        // The fragment shader is only used when the normal or velocity prepass is enabled or the material uses an alpha mask
         let fragment = if key.mesh_key.contains(MeshPipelineKey::NORMAL_PREPASS)
+            || key.mesh_key.contains(MeshPipelineKey::VELOCITY_PREPASS)
             || key.mesh_key.contains(MeshPipelineKey::ALPHA_MASK)
         {
+            shader_defs.push("PREPASS_FRAGMENT".into());
```

Comment:
Why is this necessary? Isn't it enough to simply set the fragment shader to `None`?

Reply:
In one of the pbr shaders the prepass imports, we need to #ifdef out a function if the prepass fragment shader (e.g., normal or velocity) is enabled. 

Also, we define the fragment shader fields output dynamically based on ifdefs, but you can't have an empty struct, which would happen when neither velocity or normals are enabled.

For these reasons, we need an ifdef representing whether the prepass fragment shader will be used or not.